controllers/constant/commonservice.go

```diff
@@ -22,8 +22,6 @@ kind: Subscription
 metadata:
   name: ibm-common-service-operator
   namespace: {{ .MasterNs }}
-  annotations:
```

Comment:
@ZhuoxiLi Please notice we need to remove the version from ODLM and namespacescope operator as well